ngraph/core/src/op/atanh.cpp

```diff
@@ -70,7 +74,6 @@ namespace atanhop
 
 bool op::v3::Atanh::evaluate(const HostTensorVector& outputs, const HostTensorVector& inputs) const
 {
-    bool rc = false;
-    NGRAPH_OP_SCOPE(v3_Atanh_evaluate) { rc = atanhop::evaluate_atanh(inputs[0], outputs[0]); }
-    return rc;
+    NGRAPH_OP_SCOPE(v3_Atanh_evaluate) { return atanhop::evaluate_atanh(inputs[0], outputs[0]); }
```

Comment:
can we have like:
```c++
NGRAPH_SCOPED_CODE(v3_Atanh_evaluate, { return atanhop::evaluate_atanh(inputs[0], outputs[0]); })
```
which is :
```
#if LABEL_IS_HERE
#define NGRAPH_SCOPED_CODE(label, code) code
#else
#define NGRAPH_SCOPED_CODE(label, code) throw ngraph_error(TOSTR(label) " is disabled!");
#endif
```
// just a pseudo code

Reply:
Fixed